### DISCUSSION TITLE
I built some Angular date-fns pipes

### DISCUSSION THREAD

**Discussion by Aaron-Sterling** [js-fns/date-fns#2842] [Show and tell]
Hi folks, wanted to drop a line to let you know I published a few Angular pipes to npm that I built with date-fns. I was working with Ionic, and the date-fns functions were more consistent across platforms than the default Angular date pipe.

Thank you for all your great work!

Link: https://www.npmjs.com/package/ng-datefns-pipes

**Comment by tan75** [comment]
hi @Aaron-Sterling 
Thanks for this!
I am going to convert this to a discussion.
